4_de_analysis/0_download_counts.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download pan-cancer RNA-seq read counts data from UCSC Xena Browser\n",
+    "\n",
+    "The other RNA-seq data we downloaded in `0_data_download` contains pre-processed RPKM values. In order to do differential expression analysis, most methods recommend using count data, or something similar such as [RSEM expected counts](https://support.bioconductor.org/p/90672/#90678) (which is what we'll download here).\n",
```

Comment:
Not sure if you want to add a note for yourself, that the reason is because some methods, like DESeq and I think edgR have their own processing/normalization that they'll do when they perform DE analysis